scripts/Password.vue

```diff
@@ -0,0 +1,52 @@
+<script setup>
+import { translation } from './i18n.js';
+import { ref, watch } from 'vue';
+
+const emit = defineEmits(['update:modelValue']);
+
+const password = defineModel('password', {
+    default: '',
+});
+const show_toggle = defineModel('show_toggle', {
+    default: true,
+});
+const is_visible = defineModel('is_visible', {
+    default: false,
+});
+const passwordInput = ref(null);
+
+function focus() {
+    passwordInput?.value?.focus();
+}
+
+defineExpose({ focus });
+
+watch(password, (newVal) => {
+    emit('update:modelValue', newVal);
+});
```

Comment:
```suggestion
```
This should already be handled by `defineModel`

Reply:
When removed, the password doesn't seem to pass through to the 'parent' component.

Reply:
It's because `defineModel` has `password` as name, but you're using the default `v-model` in props. Either remove the name or change to `v-model:password`